examples/quick_start/scripts/ranking/ranking.py

```diff
@@ -251,7 +251,7 @@ def train_eval_stl(self, model):
                 epochs=self.args.epochs,
                 batch_size=self.args.train_batch_size,
                 steps_per_epoch=self.args.train_steps_per_epoch,
-                shuffle=False,
+                shuffle=self.args.shuffled_train,
```

Comment:
@gabrielspmoreira I am curious if shuffle arg in model.fit works as expected? meaning does it really shuffle data or not, if it is not explicitly defined in the Loader? thanks.

Reply:
I think you are right. Shuffle here only applies for Dataset. Let me change it to set shuffle in the loader